docs/usage.md

```diff
@@ -1,61 +1,11 @@
 # nf-core/cageseq: Usage
```

Comment:
Is there a reason why this page is mostly empty?

I am also confused for example why `--fasta` was removed, but it remains in your `--help` message and you do use it in the pipeline (https://github.com/nf-core/cageseq/blob/1c6870f37e5dedd66d4925c0130ff801ef697592/main.nf#L423)?

Reply:
Since we have now the `nextflow_schema.json` rendered as parameter docs, I moved all of them in there (which was the reason why I implemented the feature, so I didn't have to repeat myself here 🙂).

Reply:
Oh wow, I jut saw that! I didn't realise that had actually gone in, in the end! 

Fair enough. It's a shame it can't back-post (so to say). 

That said, personally whenever I am working with a bioinformatic tool I'll always check the github rather than institutional webpage as I assume the former is more up-to-date... also it means you won't have your docs 'easily accessible' in the Zenodo archive when you publish as it might not be clear from someone who a long time in the future might have to download from there. 

Therefore, I dunno what others in nf-core would think about leaving the Markdown essentially completely empty here... @drpatelh @MaxUlysse @apeltzer @ewels thoughts?

Reply:
Actually, maybe you could also automatically duplicate to create the `docs/usage.md` -  when you update the JSON you also create a `.md` version that gets placed under `docs/`?

Reply:
btw. the usage.md is not completely empty, the things from the template about more general usage patterns are still there. 
If a tool has a readthedocs page, are you still reading their files on github instead?

Going through the eager PR I am actually even stronger in favor of not having the parameter docs duplicated. I find it already hard to maintain and update a >4000 line document, but keeping the information in sync in two places (usage.md and nextflow_schema.json) makes it even easier to introduce inconsistencies, imo. 

I don't think we need to render it out for github, but I think we should mention on top of usage.md, that the full documentation can be found on nf-co.re (as @ewels already mentioned somewhere).

Reply:
Sorry, yes - to be clear I didn't mean completely empty of everything but of anything CAGESeq related.

> If a tool has a readthedocs page, are you still reading their files on github instead?

Sometimes, actually yes, and as sometimes I want to investigate the docs in parallel to the code so it's more easily accessible.

> but keeping the information in sync in two places (usage.md and nextflow_schema.json)

You're right about that, which is why I suggested also making an automated `.md` export on each `finished` press of `schema build`.

My reasoning is here:

1) not to be a negative nelly, but GitHub has far more money/infrastructure to keep a project going for much longer than `nf-co.re` which is essentially reliant on volunteers and academic (i.e. ad-hoc ) funding.
2) Related is that for long-term archiving (e.g. with Zenodo), having the docs hidden in some unusual file name will not help longevity. If I was hunting for documentation I wouldn't look for a file called `nextflow_schema.json`, but `.md` is ubiquitous. 
3) GitHub is the primary place where people will look for programs, they would only find the nf-co.re website later, so it doesn't help people to evaluate if they want to use a pipeline if they can't find docs where they first find the pipeline - and having to go to an unfamiliar website where they don't know the interface might put people off investigating further.

I think the strength of nf-co.re is when people want a quick reference when they are familiar the pipeline because of the ToC sidebar.

But I guess this isn't a discussion for cageseq now, so maybe I'll push to slack ;).